### PR TITLE
DOC-9645: New error messages and deprecation notices for earlier rest…

### PR DESCRIPTION
**Pull request by tonyjhillman**
… cert mgmnt apis

### CODE REVIEW

File: modules/rest-api/pages/deprecated-security-apis/upload-retrieve-root-cert.adoc [hunk 1]
```diff
@@ -89,6 +93,9 @@ An incorrectly specified IP address or domain name causes the attempted connecti
 
 If `GET /pools/default/certificates`, which is deprecated in version 7.1, is used on a cluster with more than one root CA, `400 Bad Request` is returned, with the following error message: `deprecated, please use /pools/default/trustedCAs`.
 
+If the `POST` method and `/controller/uploadClusterCA` URI (which are deprecated in 7.1) is used from a host other thatn _localhost_, `400 Bad Request` is returned, with the following error message:
```
Comment (timofey-barmin) [inline modules/rest-api/pages/deprecated-security-apis/upload-retrieve-root-cert.adoc]: typo: thatn

File: modules/rest-api/pages/deprecated-security-apis/upload-retrieve-root-cert.adoc [hunk 2]
```diff
@@ -27,12 +31,12 @@ GET /pools/default/certificate
...
 However, this default setting can be changed, by means of the `POST` method and `/settings/security/allowNonLocalCACertUpload` URI, with the `-d false` flag: this also can be used on _localhost_ only.
 The status of the current setting can be retrieved with the `GET` method and `/settings/security/allowNonLocalCACertUpload` URI.
 
 The `GET` method and `/pools/default/certificate` URI are deprecated in 7.1.
-After the cluster has been upgraded to 7.1, they continue to return the certificate that was used prior to upgrade, until at least one new root certificate is uploaded: following which they always return `400 Bad Request`, with the message `deprecated, please use /pools/default/trustedCAs`.
+After the cluster has been upgraded to 7.1, they continue to return the certificate that was used prior to upgrade, until at least one new root certificate is uploaded using the 7.1 API `POST /node/controller/loadTrustedCAs` (as described in xref:rest-api:load-trusted-cas.adoc[Load Root Certificates]): following which they always return `400 Bad Request`, with the message `this API is disabled, please use GET /pools/default/trustedCAs, see documentation for details`.
```
Comment (timofey-barmin) [inline modules/rest-api/pages/deprecated-security-apis/upload-retrieve-root-cert.adoc]: After rereading I realized that one more small correction is needed.

> they continue to return the certificate that was used prior to upgrade

It actually now returns the most recent certificate that was uploaded by `POST /controller/uploadClusterCA`.
So incorrectness of current version is in the fact that we can use `POST /controller/uploadClusterCA` **after** the upgrade, and `GET /pools/default/certificate` will return this uploaded cert in this case.

File: modules/rest-api/pages/deprecated-security-apis/upload-retrieve-root-cert.adoc (hunk 1 above)
Comment (tonyjhillman) [reply modules/rest-api/pages/deprecated-security-apis/upload-retrieve-root-cert.adoc]: Thanks. Fixed.

File: modules/rest-api/pages/deprecated-security-apis/upload-retrieve-root-cert.adoc (hunk 2 above)
Comment (tonyjhillman) [reply modules/rest-api/pages/deprecated-security-apis/upload-retrieve-root-cert.adoc]: I have rewritten this accordingly.

**Review by timofey-barmin — APPROVED** [review]

**Pull request merged**